tests/testthat/test-read_redcap.R

```diff
@@ -565,3 +565,48 @@ test_that("read_redcap works for a large sparse database", {
     sapply(class) %>%
     expect_equal(expected_col_types)
 })
+
+test_that("read_redcap works with non-longitudinal Data Access Groups", {
+  httptest::with_mock_api({
```

Comment:
Do we want to check that a non-DAG database doesn't return the `redcap_data_access_group` column?

Reply:
Sure!

Reply:
I think this still needs to be done?

Reply:
Odd, somehow it seems to have gotten lost from [this commit (line628)](https://github.com/CHOP-CGTInformatics/REDCapTidieR/pull/153/commits/e2dfb64825fa5546b398676ae4f4e817868ca166). I'll add it back in.

Reply:
Ah nvm, this was me. Added it back in but needed a slight rework for the check.